src/config/ILessProps.ts

```diff
@@ -1,3 +1,4 @@
 export interface ILessProps {
   plugins: Array<any>;
+  options: any;
```

Comment:
Can you make typings for this?

Reply:
We could use Less typings https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/less/index.d.ts
But how would I import typings for another project?